cmd/vulcan-gitleaks/manifest.toml

```diff
@@ -0,0 +1,5 @@
+Description = "Finds potential secrets in source code from a Git repository using gitleaks"
+Timeout = 900 # 15 minutes
+AssetTypes = ["GitRepository"]
+RequiredVars = []
```

Comment:
I think we are missing the required vars to be able to clone the repo:
```RequiredVars = ["GITHUB_ENTERPRISE_ENDPOINT", "GITHUB_ENTERPRISE_TOKEN"]```

Reply:
Yes, I added them here [ec07e44](https://github.com/adevinta/vulcan-checks/pull/185/commits/ec07e443a7a05c55ec762fc6ffe12642f82d8256).